Doc/library/os.rst

```diff
@@ -3951,7 +3951,7 @@ to be ignored.
 
    .. note::
 
-      The standard way to exit is ``sys.exit(n)``.  :func:`_exit` should
+      The standard way to exit is :func:`sys.exit`.  :func:`_exit` should
```

Comment:
I think the new version sounds a bit weird. I would either use:
```suggestion
      The standard way to exit is :func:`sys.exit(n) <sys.exit>`.  :func:`_exit` should
```
or:
```suggestion
      The standard way to exit is calling :func:`sys.exit`.  :func:`_exit` should
```

Reply:
Agreed, I guess, particularly with the first (which preserves the current text).